GNUmakefile

```diff
@@ -48,7 +43,7 @@ attach:
 
 acceptance: fmtcheck
 	export TF_ACC=true && \
-		go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -p 1 -parallel 20 -timeout 20m ./pkg/...
+		go test -v -p 1 -parallel 20 -timeout 20m ./pkg/...
```

Comment:
I really at this point would like to print out the version when the TF provider runs.

Reply:
It does already but I need to revert this so the correct version is logged.